src/psyclone/psyir/nodes/__init__.py

```diff
@@ -31,7 +31,8 @@
 # ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 # POSSIBILITY OF SUCH DAMAGE.
 # -----------------------------------------------------------------------------
-# Author: S. Siso, STFC Daresbury Lab
+# Author S. Siso, STFC Daresbury Lab
+# Author J. Henrichs, Bureau of Meteorology
```

Comment:
For the record - I did not try to sneakily claim authorship ... I did create this function first, then Sergi's branch was merged in. I've also renamed 'modification' to 'modified', since this is what we use elsewhere.

Reply:
Oh yes, I missed that, apologies. Please keep yourself in the author list (if you can be bothered).